build-logic/buildquality/src/main/kotlin/gradlebuild/configcachereport/tasks/VerifyDevWorkflow.kt

```diff
@@ -22,8 +22,10 @@ import org.gradle.api.tasks.InputDirectory
 import org.gradle.api.tasks.PathSensitive
 import org.gradle.api.tasks.PathSensitivity
 import org.gradle.api.tasks.TaskAction
+import org.gradle.work.DisableCachingByDefault
 
 
+@DisableCachingByDefault(because = "Not worth caching")
 abstract class VerifyDevWorkflow : DefaultTask() {
```

Comment:
We could actually cache this if it produced a file output, no? Or if we had non-file output handling. So it's not cacheable because it (currently) has no cacheable output.

Reply:
It doesn't have an output, though even if it would, it is not worth caching.